downloads/downloader.go

```diff
@@ -20,11 +22,32 @@ type DatasetClient interface {
 	GetVersion(ctx context.Context, userAuthToken, serviceAuthToken, downloadServiceToken, collectionID, datasetID, edition, version string) (m dataset.Version, err error)
 }
 
+// ImageClient is an interface to represent methods called to action on the image api
+type ImageClient interface {
+	GetImage(ctx context.Context, imageID string) (m image.Image, err error)
```

Comment:
if Image API is running in `publishing` mode we will need to provide the tokens here (and modify dp-api-clients-go to pass them as headers)